migration/migration.go

```diff
@@ -410,6 +410,9 @@ func GetMigrations() Migrations {
 	// Version 90
 	m = append(m, steps{ExecuteSQLFile("090-queries-version.sql")})
 
+	// Version 90
+	m = append(m, steps{ExecuteSQLFile("091-comments-child-comments.sql")})
```

Comment:
A test is required to ensure the new column exists. See `migration/migration_blackbox_test.go`

Reply:
Added.